mix/CodeModel.cpp

```diff
@@ -197,6 +197,8 @@ CodeModel::~CodeModel()
 	stop();
 	disconnect(this);
 	releaseContracts();
+	if (m_gasCostsMaps)
+		m_gasCostsMaps->deleteLater();
```

Comment:
There will be no event handlers after this, should be deleted immediatelly i think